### ISSUE TITLE
How do I verify that my disc/my Wii/flash storage device is broken and not Cleanrip

### ISSUE THREAD

**Issue by treymerkley** [emukidid/cleanrip#56]
I've been trying to dump a few discs, but they all seem to be failing. Some discs have light scratches, others are apparently perfectly clean, and they all boot in the Wii. 

The way it usually happens is that I run Cleanrip, and as the data transfer rate declines, the time taken to finish increases, until Cleanrip encounters an error and closes in ten seconds.

Is there a log or something I can look at to figure out what the problem is? I figure it's on my end, but it would be nice to be able to verify.

**Comment by emukidid** [comment]
Based on the 10 seconds -> exit logic you're running into, it sounds like you're experiencing a write error instead of a read error. What kind of storage (SD or DVD) and file system are you writing to?

Try dumping a GameCube disc if you have any as they're under 2GB and see how that goes.

**Comment by billywade** [comment]
I was dumping GameCube games is the thing. I'm dumping to a USB flash drive, 16Gb, NTFS. Do I need to reduce the partition size or something?

**Comment by Bombastisch** [comment]
Try FAT 32. I think I have had a similar problem once. Using FAT32 fixed the problem for me.

**Comment by billywade** [comment]
Just tried FAT32, but it's still dropping out. However, it is definitely a write error, or at least that's the label.

**Comment by Bombastisch** [comment]
I tried several USB devices but only some work. I think it is a driver bug, because USB Loader GX is known to work only with certain USB sticks, too. If they work or not seems really random. I tried a new USB 3.0 stick and it worked great:

![grafik](https://user-images.githubusercontent.com/28686186/51489659-407b1f00-1da9-11e9-80e8-cd4878e7b5af.png)

I also tried an USB 2.0 but that one did not work...

SD on the other hand seems to work all the time.
